compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DocumentationAnalyzer.java

```diff
@@ -231,6 +231,9 @@ public void visit(BLangTypeDefinition typeDefinition, AnalyzerData data) {
 
             for (SimpleVariableNode field : fields) {
```

Comment:
This can be 
```suggestion
            for (BLangSimpleVariable field : fields) {
```

right?

Then the logic in L234-L236 becomes the same as L219-L221, we can probably extract it out to a method.

Reply:
Updated 010fc302bc4f763c00756742ed2b769746efa17c